Makefile

```diff
@@ -123,8 +123,19 @@ clientset-generate: ## Generate client-go clientset, listers and informers.
 pkg/scalers/liiklus/LiiklusService.pb.go: hack/LiiklusService.proto
 	protoc -I hack/ hack/LiiklusService.proto --go_out=plugins=grpc:pkg/scalers/liiklus
 
+.PHONY: mockgen-gen
+mockgen-gen: mockgen pkg/mock/mock_scaling/mock_interface.go pkg/mock/mock_scaler/mock_scaler.go pkg/mock/mock_scale/mock_interfaces.go pkg/mock/mock_client/mock_interfaces.go pkg/scalers/liiklus/mocks/mock_liiklus.go
```

Comment:
`mockgen-gen` is not a great name, but not sure how else to call it to maintain consistency.

Reply:
Agree, but I can think of any better. Let's keep it this way :)